next-i18next.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require("path");
+
+const isProduction = process.env.NODE_ENV === "production";
+const localePath =
+    typeof window === "undefined"
+        ? path.resolve("./public/static/locales")
+        : "/static/locales";
+
+module.exports = {
+    debug: !isProduction,
```

Comment:
Turning on the debug flag in dev helps catch i18n key typos and missing i18n messages, which already helped me catch these: 8415b299a226ab229030dbe378e4c2e037e005a8